suite-native/assets/src/components/DiscoveryAssetsLoader.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { ListItemSkeleton, HStack, Text } from '@suite-native/atoms';
+import { Icon } from '@suite-common/icons';
+import { useTranslate } from '@suite-native/intl';
+
+type DiscoveryAssetsLoaderProps = {
+    isListEmpty: boolean;
+};
+
+export const DiscoveryAssetsLoader = ({ isListEmpty }: DiscoveryAssetsLoaderProps) => {
+    const { translate } = useTranslate();
+
+    const discoveryProgressText = translate(
+        isListEmpty
+            ? 'assets.dashboard.discoveryProgress.1'
```

Comment:
nit for discussion: woulnd't go with numbered translation strings as that's not saying anything. Guess it could be described better and just use some nested naming

Reply:
I would prefer not to use numbers because now you have no idea what that number means

Reply:
Ah, ok, It was misunderstanding then. Number removed in 019dcf358